cfgov/cfgov/settings/base.py

```diff
@@ -688,6 +691,13 @@
 
     # Test financial well-being hub pages on Beta
     'FINANCIAL_WELLBEING_HUB': [('environment is', 'beta')],
+
+    # Test migrated HMDA content pages. Delete after HMDA content launch
+    'HMDA_LEGACY_REVIEW': [{'condition': 'site',
+        'value': 'content.consumerfinance.gov:80'}],
```

Comment:
At first I was questioning if this URL should appear on GH but it's elsewhere in the repo so 🤷‍♂️ .

Reply:
Theoretically this condition could be created in the admin on content rather than hard-coded.

I do wonder — I don't think we still have a separate Wagtail `Site` for content, do we?

Reply:
I think you're right, Will. There's a Sharing Site set up at that subdomain, but I assume the site condition doesn't look at that. Even if it did, the port is 443.

Reply:
An alternate way to do this would be to use `'environment is': 'content'`, which would match any hostnames that you happened to hit the content server with, and would also make it easier to test locally with, e.g.

```sh
$ DEPLOY_ENVIRONMENT=content ./runserver.sh
```

Reply:
Thanks for the feedback. I was patterning my flag condition after other ones that I saw in Wagtail, but I'll use one of the corrected versions you mentioned instead. I'll set up the condition using the Wagtail admin instead of in code.